ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/ContextExtensions.kt

```diff
@@ -14,3 +18,18 @@ internal fun Context.getActivity(): Activity? {
     }
     return null
 }
+
+private const val MAX_CACHE_SIZE_BYTES = 100 * 1024 * 1024L // 100 MB
+
+@Composable
+@ReadOnlyComposable
+internal fun Context.getRevenueCatUIImageLoader(): ImageLoader {
+    return ImageLoader.Builder(this)
+        .diskCache {
+            DiskCache.Builder()
+                .directory(cacheDir.resolve("revenuecatui_cache"))
```

Comment:
This is so we don't share the same folder as the app developer in case they also use `Coil` to load images.